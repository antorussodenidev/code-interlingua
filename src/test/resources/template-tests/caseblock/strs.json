{
  "type": "java.lang.String",
  "cases": [
    {
      "value": "foo",
      "stmts": [
        { "nodeType": "Statement", "code": "{ foo(); }" }
      ]
    },
    {
      "value": "bar",
      "stmts": [
        { "nodeType": "ExpressionStatement", "code": "bar();" }
      ]
    }
  ],
  "defaultCase": {
    "stmts": [
      "baz();"
    ]
  }
}
