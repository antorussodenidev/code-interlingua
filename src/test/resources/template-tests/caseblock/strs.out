package foo;

class C {
  C(java.lang.String x) {
    switch (x) {
    case "foo":
      {
        foo();
      }
      break;
    case "bar":
      bar();
      break;
    default:
      { baz(); }
    }
  }

}
